compiler/src/Language/Bond/Codegen/Java/Class_java.hs

```diff
@@ -535,6 +599,8 @@ public class #{typeNameWithParams declName declParams}#{maybe interface baseClas
     }
     #{bondTypeDescriptorInstanceVariableDecl}
 
+    #{ifThenElse (null declParams) (javaNativeSerializationGlue declName) javaNativeSerializationUnimpl}
```

Comment:
>#{ifThenElse (null declParams) (javaNativeSerializationGlue declName) javaNativeSerializationUnimpl} [](start = 4, length = 100)

Pls add support for generic types.

Reply:
Added a byte in front of all of this to represent the number of type parameters, which is a fixed 0 for concrete types.